tests/test_sft_trainer.py

```diff
@@ -423,6 +423,25 @@ def test_data_collator_completion_lm(self):
         result_text = self.tokenizer.decode(batch["input_ids"][0, last_pad_idx + 1 :])
         self.assertTrue(result_text == "I have not been masked correctly.")
```

Comment:
`assertEqual` is better here for useful test output. Otherwise you will get "True is not False" or something like that in case of error.